micronota/parsers/cmscan.py

```diff
@@ -0,0 +1,413 @@
+   “seq to” value, else it will be greater than or equal to it.
+11. trunc
+   Indicates if this is predicted to be a truncated CM hit or not. This will be
+   “no” if it is a CM hit that is not predicted to be truncated by the end of
+   the sequence, “5’ ” or “3’ ” if the hit is predicted to have one or more 5’
+   or 3’ residues missing due to a artificial truncation of the sequence, or
+   “5’&3”’ if the hit is predicted to have one or more 5’ residues missing and
+   one or more 3’ residues missing. If the hit is an HMM hit, this will always
+   be ’-’.
+12. pass
+   Indicates what “pass” of the pipeline the hit was detected on. This is
+   probably only useful for testing and debugging. Non-truncated hits are found
+   on the first pass, truncated hits are found on successive passes.
+13. gc
+   Fraction of G and C nucleotides in the hit.
+14. bias
+   The biased-composition correction: the bit score difference contributed by
+   the null3 model for CM hits, or the null2 model for HMM hits. High bias
+   scores may be a red flag for a false positive. It is difficult to correct
+   for all possible ways in which a nonrandom but nonhomologous biological
+   sequences can appear to be similar, such as short-period tandem repeats, so
+   there are cases where the bias correction is not strong enough (creating
+   false positives).
+15. score
+   The score (in bits) for this target/query comparison. It includes the
+   biased-composition correction (the “null3” model for CM hits, or the “null2”
+   model for HMM hits).
+16. E-value
+   The expectation value (statistical significance) of the target. This is a
+   per query E-value; i.e. calculated as the expected number of false positives
+   achieving this comparison’s score for a single query against the search
+   space Z. For cmsearch Z is defined as the total number of nucleotides in the
+   target dataset multiplied by 2 because both strands are searched. For cmscan
+   Z is the total number of nucleotides in the query sequence multiplied by 2
+   because both strands are searched and multiplied by the number of models in
+   the target database. If you search with multiple queries and if you want to
+   control the overall false positive rate of that search rather than the false
+   positive rate per query, you will want to multiply this per-query E-value by
+   how many queries you’re doing.
+17. inc
+   Indicates whether or not this hit achieves the inclusion threshold: ’!’ if
+   it does, ’?’ if it does not (and rather only achieves the reporting
+   threshold). By default, the inclusion threshold is an E-value of 0.01 and
+   the reporting threshold is an E-value of 10.0, but these can be changed with
+   command line options as described in the manual pages.
+18. description of target
+   The remainder of the line is the target’s description line, as free text.
+
+
+Format Support
+--------------
+**Has Sniffer: No**
+
```

Comment:
Don't forget to include the other format supports.

I'd refere to [here](https://github.com/biocore/micronota/blob/master/micronota/parsers/sam.py#L42)